docs/angular-testing-library/api.mdx

```diff
@@ -395,20 +395,39 @@ properties that are not defined are cleared.
 
 ```typescript
 const {rerender} = await render(Counter, {
-  componentProperties: {count: 4, name: 'Sarah'},
+  componentInputs: {count: 4, name: 'Sarah'},
```

Comment:
Is there an implementation of the `Counter` component somewhere in this page? I wasn't able to find it. Maybe it will help to add it?

Reply:
No, the implementation is "hidden" inside of the ATL repo.
I'll leave this for now. This comment makes me think if I should add a link to a working example (most of these already exist within ATL).
What's your opinion?

Reply:
For the `Counter` component? Isn't it just for the example? I think it might help to have it here but if the component is big then a link might also work.